packages/forms/src/UIForm/fields/Code/Code.component.test.js

```diff
@@ -98,7 +103,7 @@ describe('Code field', () => {
 	it('should focus on the wrapping div on double esc keydown', () => {
 		// given
 		const containerId = 'myCodeWidget-editor-container';
-		const wrapper = mount(<Code.WrappedComponent {...props} />);
+		const wrapper = mount(<Code {...props} />, { attachTo: document.body });
```

Comment:
Need to mount on document.body with talend/script jest/enzyme version, or the document.activeElement is not set.